app/services/idv/steps/doc_auth_base_step.rb

```diff
@@ -183,6 +183,12 @@ def mobile?
         client.device_type != 'desktop'
       end
 
+      def log_document_error(get_results_response)
+        return unless get_results_response.class == Acuant::Responses::GetResultsResponse
```

Comment:
Direct class checks are bad IMO. I would do either:

- a subclass check (`.is_a?(Acuant::Responses::GetResultsResponse)`
- a `.respond_to?(:result_code)` check

Reply:
There should have been one response class.  It took me a while to figure out they were different.

Reply:
I'm fearful if I use `result_code` then I'd have to check `name` afterwards too

Reply:
`is_a` doesn't buy much but I'd be happy to use that since it's an improvement